tests/integration/kubernetes/gha-run.sh

```diff
@@ -71,7 +71,7 @@ function deploy_kata() {
     sed -i -e "s|quay.io/kata-containers/kata-deploy:latest|${DOCKER_REGISTRY}/${DOCKER_REPO}:${DOCKER_TAG}|g" "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml"
 
     # Enable debug for Kata Containers
-    yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[1].value' "\"yes\""
+    yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[1].value' --tag !!str "true"
```

Comment:
Running on bash, I had to pass '!!str' because !! is going to eval to the last command in the bash's history database. Does it work fine within this script?

Reply:
Oh, interesting.  Let me fix it.